qa/common/rgw.sh

```diff
@@ -26,6 +26,21 @@ EOF
   _run_test_script_on_node $TESTSCRIPT $RGWNODE
 }
 
+function rgw_validate_system_user {
+  #
+  # prove the system user "admin" was really set up
+  #
+  local TESTSCRIPT=/tmp/rgw_validate_system_user.sh
+  local RGWNODE=$(_first_x_node rgw)
+  cat << EOF > $TESTSCRIPT
+set -ex
+trap 'echo "Result: NOT_OK"' ERR
+radosgw-admin user info --uid=admin | grep system | grep -q true
```

Comment:
I guess this won't produce any output for the log? Could we run it once without the greps, and then with?

Reply:
Sure

Reply:
done!